packages/react/src/patterns/sections/CardSection/CardSectionImages.js

```diff
@@ -3,24 +3,27 @@ import PropTypes from 'prop-types';
 import React from 'react';
 
 /**
- * ImageCards pattern it is Cards with images
+ * CardSectionImages pattern it is Cards with images
  *
  * @param {object} props props object
  * @param {Array} props.cards cards array with title, and cards properties
  * @returns {object} JSX Object
  */
-const ImageCards = props => {
-  return <CardSection {...props} />;
+const CardSectionImages = props => {
+  const cardsWithImages = props.cards.filter(
+    card => card.image && card.eyebrow
+  );
+  return <CardSection {...(cardsWithImages && { ...props })} />;
 };
 
-ImageCards.propTypes = {
+CardSectionImages.propTypes = {
   theme: PropTypes.string,
-  title: PropTypes.string,
+  heading: PropTypes.string.isRequired,
   cards: PropTypes.arrayOf(
     PropTypes.shape({
```

Comment:
Might make sense to use `exact` instead of `shape`.

Reply:
okay